python/cuml/cuml/__init__.py

```diff
@@ -62,9 +62,7 @@
     from cuml.linear_model.mbsgd_regressor import MBSGDRegressor
 
     from cuml.manifold.t_sne import TSNE
-    from cuml.metrics.accuracy import accuracy_score
-    from cuml.metrics.cluster.adjusted_rand_index import adjusted_rand_score
-    from cuml.metrics.regression import r2_score
+    from cuml.metrics import accuracy_score, r2_score, adjusted_rand_score
```

Comment:
Maybe more of a note for later, but should these things appear in the main cuml namespace? That seems totally weird, but then it seems like all these imports where put here for a reason. Should I make a new issue or is there a short answer to this question?

Reply:
I have mixed thoughts on that, but agree a larger discussion would be worth it - can you open a new issue please with your initial thoughts?

Reply:
I created https://github.com/rapidsai/cuml/issues/6411 .